lib/src/native/realm_core.dart

```diff
@@ -1345,6 +1344,11 @@ class _RealmCore {
     return userId;
   }
 
+  AppHandle userGetApp(UserHandle userHandle) {
+    // TODO: we don't have an API to get the app for a user - https://github.com/realm/realm-core/issues/5478
+    return AppHandle._(nullptr);
```

Comment:
If we are going to merge this, we should be throwing an Error here. As in not implemented or something, cause now this will crash  the app most probably.

Reply:
This just got merged in Core, so I'll pick it up and implement before merging this PR.

Reply:
I am updating Core in a branch. I could handle this here as well.

Reply:
If you update in master, I'll just rebase this once your PR gets merged.

Reply:
its merged